web/src/components/AppFooter.vue

```diff
@@ -5,41 +5,90 @@
     app
     absolute
   >
-    <v-container>
+    <v-container
+      fluid
+    >
       <v-row no-gutters>
         <v-col
           cols="12"
-          justify="center"
-          align="center"
+          md="4"
+          xs="12"
         >
-          <a
-            class="footer-link"
-            target="_blank"
-            href="https://github.com/WWCodeTokyo/speak-her-db"
-          >{{ $t('footer.dateString') }}</a>
+          <v-row :justify="$vuetify.breakpoint.mdAndUp ? left : center">
```

Comment:
This line isn't working the way I expected 😓

Reply:
You don't need this inner row inside each col, I explained it better [here](https://github.com/WWCodeTokyo/speak-her-db/pull/179#discussion_r499583984)

Reply:
The layout was very different when I added vs. removed them.